### ISSUE TITLE
Map render is out of date

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#34311]
(WEB REPORT BY: inorien REMOTE: 206.221.180.138:7777)
# Revision
c9830afd36a3223589265e1db302ea1f8051558d

# Description
Map render for Box is out of date, haven't yet verified others

# Steps to Reproduce
Be on Box
Go to top right and click Box Station
look above tool storage and compare it with what's really there

# What you Expected
It be like it is

# What Actually Happened
It don't be like it is

**Comment by DorainGrey** [comment]
Yes admins need to generate them during 0 pop but no one bothers.

**Comment by SECBATON-GRIFFON** [comment]
yeah the ones on the website are all painfully out of date
